### ISSUE TITLE
Not update after charging

### ISSUE THREAD

**Issue by equiman** [Fukkei/zmk-split-battery#4]
We I connect the keyboard to the USB-c it stops reporting the percentage. 

When finish charging (orange led turned off) I need to open the panel disconnect the device and reconnect again to show the updated percentage again.

**Comment by Fukkei** [comment]
Yeah, its probably because BatteryMonitor does not track BLE device connect/disconnect events, this needs to be implemented

**Comment by equiman** [comment]
Ok I'm going to report it on the PR
https://github.com/zmkfirmware/zmk/pull/1243

Thanks!

**Comment by Fukkei** [comment]
No, the problem is with my code)
